src/main/scala/diplomaticobjectmodel/model/OMErrorDevice.scala

```diff
@@ -0,0 +1,11 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.diplomaticobjectmodel.model
+
+
+case class OMErrorDevice(
+  memoryRegions: Seq[OMMemoryRegion],
+  interrupts: Seq[OMInterrupt],
+  specifications: Seq[OMSpecification],
```

Comment:
What is this `specifications` field for? You seem to only set it to `Nil`. We should just remove it if there's not defined meaning to it.

Reply:
This is probably due to my PR, which was a blind copy-paste of some other `OM*` from some other device.